src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/internal/KoverReportExtension.kt

```diff
@@ -94,6 +94,8 @@ internal open class KoverHtmlReportConfigImpl @Inject constructor(private val ob
 
     override var title: String? = null
 
+    override var charset: String? = null
```

Comment:
Please also override this property in `KoverHtmlReportConfigWriter`

Reply:
oh i missed it! thank you